server/src/main/kotlin/org/jetbrains/bsp/bazel/server/bsp/managers/BazelExternalRulesQuery.kt

```diff
@@ -41,8 +49,7 @@ class BazelWorkspaceExternalRulesQueryImpl(private val bazelRunner: BazelRunner)
   private fun Document.calculateEligibleRules(): List<String> {
     val xPath = XPathFactory.newInstance().newXPath()
     val expression =
-      "/query/rule[contains(@class, 'http_archive') and " +
-        "(not(string[@name='generator_function']) or string[@name='generator_function' and contains(@value, 'http_archive')])" +
```

Comment:
I would prefer to revert this change as it will introduce a lot of false positives.